signin/style.css

```
@@ -0,0 +1,153 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Pretendard', sans-serif;
+  width: 100%;
+  margin: 0;
+  min-height: 100vh;
+  background: #F0F6FF;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.login {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 30px;
+}
+
+header {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  gap: 30px;
+}
+
+section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 10px;
+}
+
+footer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 10px;
+}
+
+.header-logo {
+  width: 210.58px;
+  height: 38px;
+}
+
+.header-signup {
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  color: #000;
```

```suggestion
  color: black
```
이렇게 쓰셔도 저렇게 쓰셔도 되는데 참고차 남겨요~